frontend/components/project/ProjectList.vue

```diff
@@ -131,6 +136,22 @@ export default Vue.extend({
   },
 
   methods: {
+    ...mapGetters('userState', ['getCurrentlyAllowedProjectId']),
+
+    enableProjectLink(projectId: Number) {
+      if (this.showSelect) {
```

Comment:
I think rather than using this value, its better to use `isStaff` from `mapGetters` because the `showSelect` value may change depending on the requirement, what do you think?

Reply:
noted 😅 I did this because I saw that showSelect is set on the project index page with the isStaff value and I thought it would be a bit redundant to add another getter, but I agree that it is better to use isStaff separately XD